app/views/layouts/head/_meta.html.erb

```diff
@@ -1,10 +1,24 @@
 <meta name="application-name" content="<%= t "general.project_shortname", default: APP_OPTS["project_shortname"] %>">
-  <% if @description.present? %>
+
```

Comment:
The indentation in this file wasn't a mistake. It was intended to keep the indentation in the output (browser view source) consistent, though it's kind of awkward to try to manage that. No need to revert it, just wanted to explain why the file was originally that way.

When it's up to me alone, I just strip out all of the whitespace possible in HTML output where it doesn't affect display in production env. I still leave it there in dev for debugging for invalid HTML like Karin was looking at earlier. Otherwise looking at the layout in the dev tools is nicer anyway. Then I don't see all the awkward indentation resulting from the layers of ERB layouts, views, helpers, and partials. :man_shrugging:

Reply:
Ahh okay. Since I work more with the erb templates I find it far easier to indent there than to think about how it will appear in the source.  Makes sense :)

Reply:
Yeah, that is much simpler